@here/harp-examples/resources/dayTwoFonts.json

```diff
@@ -986,29 +987,21 @@
        14
       ],
       "values": [
-       0,
        1700,
-       1700,
-       700,
-       350,
-       190,
-       150,
-       120,
-       80,
-       40,
-       25,
-       10
+       1300,
+       500,
+       220,
+       90,
+       50,
+       30,
+       20,
+       15,
+       10,
+       5,
+       2
       ]
-     }
-    }
-   },
-   {
-    "description": "borders foreground",
-    "when": "$layer == 'admin' && ($layer == 'admin')",
-    "technique": "solid-line",
-    "attr": {
-     "color": "#5C5C5C",
-     "lineWidth": {
+     },
+     "outlineWidth": {
```

Comment:
It's just discussion, by no means try to blocki patch.

I generally like this patch, but  for future designers of themes/styles/techniques i would like to define `outlineWidth` for as percentage, so in general case I don't have to interpolate at all.

Reply:
Agree that percentage is more intuitive, but don't want to change the units the designers work with with not a strong enough reason.